sass/dependencies/_breakpoints.sass

```diff
@@ -0,0 +1,10 @@
+$mediaQueries: map-get($theme, 'mediaQueries')
+
+@function media-query($name)
+  @return unquote(map-get($mediaQueries, $name)) or $name
```

Comment:
Falling back to `$name` is a convention I adopted during Scality that in retrospect causes more problems that in solves. Think we should abandon this, as it's almost never the expected the behavior.